Watchman.DomainModel/Warns/Commands/RemoveWarnEventsCommand.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.Text;
+using Watchman.Cqrs;
+
+namespace Watchman.DomainModel.Warns.Commands
+{
+    public class RemoveWarnEventsCommand : ICommand
+    {
+        public ulong? GrantorId { get; }
```

Comment:
czemu jest nullable? może być sytuacja gdzie GrantorId lub ReceiverId nie istnieje?
(według mnie nie)

Reply:
chodzi o to, że mogę chcieć usunąć albo wszystkie ostrzeżenia pewnej osoby (receiverId = 262, grantorId = null) albo usunąć wszystkie ostrzeżenia nadane przez pewną osobę (np. jak to nadawał wyrzucony moderator ktory trollował ostrzeżeniami)

Reply:
jednocześnie w handlerze jest zabezpieczenie, że nie mogą być null'em obie wartości bo to prowadziłoby do wyczyszczenia całej bazy z ostrzeżeń danego serwera